tests/FakeItEasy.Tests/Creation/DefaultFakeObjectCreatorTests.cs

```diff
@@ -142,31 +142,26 @@ public void Should_not_throw_when_unsuccessful_and_throw_on_failure_is_false()
         [Test]
         public void Should_try_with_resolved_constructors_incorrect_order()

```

Comment:
I'm not sure in what commit we'd want to fix this, and feel free to leave, but I think this should be "in_correct_order"